bring-it-up/bring-it-up
# BE#42

## Description
#### Summary:
Added keywordSearch to JSON

#### Changes:
Updated object schema and interface to reinforce types in models\counsellingService.model.ts
Updated data.json to include keywordSearch



#### How to test this PR:
- Go to postman and then run the put command to upload all the data to mongodb
- The data from the data.json should now update the database

  
## Checklist

- [ x] I performed a self-review of my code
- [ x] I ran all the tests and they were successful


shoutout to my boy @ImaadJ10 for helping me test it
